### DISCUSSION TITLE
Not getting expected quests for bicycle path surfaces

### DISCUSSION THREAD

**Discussion by ERYpTION** [streetcomplete/StreetComplete#5661] [Q&A] [answered]
I am trying to set up a profile for exclusively displaying the bicycle path surface quests (I realize that there is also the option to use the surface overlay but for the sake of simplicity I wish to only see the bicycle paths). In the quest selection the only one I have enabled is "What's the surface of the bicycle path here?"

However, before heading out I checked an area in SC against a [overpass turbo check](https://overpass-turbo.eu/s/1M68) for cycleways missing surface information. This shows several several cycleways missing surface info. However downloading data for the same area in SC doesn't get a single quest for bicycle path surface. If I turn on the surface layer in SC, I see the same missing surfaces as the overpass turbo check shows.

Why don't I get any bicycle path surface quests? I can't figure out what I am doing wrong in this setup.

![Screenshot 2024-05-27 at 17-06-54 overpass turbo](https://github.com/streetcomplete/StreetComplete/assets/80313840/8deeba1e-e5e7-4ed2-b47f-a14617a3db51)![Screenshot_20240527-165922](https://github.com/streetcomplete/StreetComplete/assets/80313840/ccd1f441-5a74-42e5-ab37-b06947a01788)![Screenshot_20240527-171228_1](https://github.com/streetcomplete/StreetComplete/assets/80313840/7b16c6c5-4d56-437e-a150-6492e58b3f83)

**Comment by FloEdelmann** [comment, marked as answer]
> I wish to only see the bicycle paths

I guess you enabled the "What’s the surface of the bicycle path here?" ([`AddCyclewayPartSurface`](https://github.com/streetcomplete/StreetComplete/blob/master/app/src/main/java/de/westnordost/streetcomplete/quests/surface/AddCyclewayPartSurface.kt)) quest then? Note that it is only asked for `segregated` paths.
The surface of non-`segregated` paths, footways and cycleways is recorded with the "What’s the surface here?" ([`AddPathSurface`](https://github.com/streetcomplete/StreetComplete/blob/master/app/src/main/java/de/westnordost/streetcomplete/quests/surface/AddPathSurface.kt)) quest.

See also <https://wiki.openstreetmap.org/wiki/StreetComplete/Quests>.

**Reply by ERYpTION** [reply]
Thanks. I didn't realize the "What’s the surface of the bicycle path here?" is only for segregated paths. Activating the other quest did the trick.
